backend/clickhouse/errors.go

```diff
@@ -164,74 +163,6 @@ func (client *Client) WriteErrorObjects(ctx context.Context, objects []*model.Er
 	return nil
 }
 
-func getErrorQueryImplDeprecated(tableName string, selectColumns string, query modelInputs.ClickhouseQuery, projectId int, groupBy *string, orderBy *string, limit *int, offset *int) (string, []interface{}, error) {
```

Comment:
Just check if these are still needed for external APIs (e.g. Grafana)?

Reply:
yeah I'm not 100% sure - there were some issues with the tests though so I'm just separating the deprecated code removal into its own PR to avoid delaying things